kubernetes-tests/src/test/java/io/fabric8/kubernetes/client/mock/StorageSpaceCrudTest.java

```diff
@@ -30,6 +30,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import static junit.framework.TestCase.assertNull;
```

Comment:
Maybe better use `org.junit.jupiter.api.Assertions.assertNull` just for consistency